keanu-project/src/main/java/io/improbable/keanu/algorithms/variational/NonGradientOptimizer.java

```diff
@@ -40,7 +40,19 @@ private void handleFitnessCalculation(double[] point, Double fitness) {
         }
     }
 
-    public double optimize(int maxEvaluations, double boundsRange, List<Vertex> outputVertices) {
+    public double optimize(int maxEvaluations,
+                           double boundsRange,
+                           List<Vertex> outputVertices){
+        double initialTrustRegionRadius = BOBYQAOptimizer.DEFAULT_INITIAL_RADIUS;
```

Comment:
Possibly put these as static variables at the top? `DEFAULT_INITIAL_TRUST_REGION_RADIUS` and `DEFAULT_STOPPING_TRUST_REGION_RADIUS`. Then we can do:

```
    public double optimize(int maxEvaluations,
                           double boundsRange,
                           List<Vertex> outputVertices){
        return optimize(maxEvaluations, boundsRange, outputVertices, INITIAL_TRUST_REGION_RADIUS, STOPPING_TRUST_REGION_RADIUS);
    }
```